### PR TITLE
[Py] Update serialization

### PR DESCRIPTION
**Pull request by tsocha**

### CODE REVIEW

File: python/ngraph/runtime.py [hunk 1]
```diff
@@ -93,9 +93,9 @@ def __call__(self, *input_values):  # type: (*NumericData) -> NumericData
         result_arr = result_arr.reshape(result_shape)
         return result_arr
 
-    def serialize(self):  # type: () -> str
+    def serialize(self, indent=0, bin_const_data=False):  # type: (size_t, bool) -> str
         """Serialize function (compute graph) to a JSON string."""
```
Comment (postrational) [inline python/ngraph/runtime.py]: Add documentation of parameters.

File: python/ngraph/runtime.py [hunk 2]
```diff
@@ -93,9 +93,9 @@ def __call__(self, *input_values):  # type: (*NumericData) -> NumericData
         result_arr = result_arr.reshape(result_shape)
         return result_arr
 
-    def serialize(self):  # type: () -> str
+    def serialize(self, indent=0, bin_const_data=False):  # type: (size_t, bool) -> str
```
Comment (postrational) [inline python/ngraph/runtime.py]: `size_t` is not defined in Python

**Comment by postrational** [comment]
Please add at least one unit test for this serialization function.

**Comment by tsocha** [comment]
@rkimballn1 how should we document parameter: binary_constant_data?

**Pull request merged**